models/articles.models.js

```diff
@@ -23,7 +23,7 @@ function selectArticleById(articleId) {
     });
 }
 
-function selectAllArticles() {
+function selectArticles(topic = "%") {
```

Comment:
Based on comments above that topics that don't exist should be a 404 you will most likely need to come back and refactor this. We don't really want to match any topic other than ones that do exist in the topics table, else why have the table 😄